<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hider</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnChoseExe">
          <property name="text">
           <string>选择程序</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveExe">
          <property name="text">
           <string>移除程序</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearExe">
          <property name="text">
           <string>清空程序</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QKeySequenceEdit" name="keySequenceEdit">
        <property name="maximumSequenceLength">
         <longLong>1</longLong>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetHotKey">
        <property name="text">
         <string>设置快捷键</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMini">
        <property name="text">
         <string>托盘</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labStatus">
        <property name="text">
         <string>未启动</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
